src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/Post.java

```diff
@@ -7,11 +7,14 @@
 
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Lists;
+import lombok.SneakyThrows;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriverException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.WebDriverException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)